### DISCUSSION TITLE
dep rollback

### DISCUSSION THREAD

**Discussion by jasperf** [deployphp/deployer#2724] [Help needed]
https://deployer.org/docs/7.x/recipe/deploy/rollback#rollback-1 is empty for now. How to rollback to a specific version again?

**Comment by antonmedv** [comment]
Try dep rollback -o rollback_candidate=123

**Reply by jasperf** [reply]
Thanks again. This worked

 ```
dep rollback -o rollback_candidate=46 prod
task rollback
[prod] Current release is 48.
[prod] Rolling back to 46 release.
[prod] rollback to release 46 was successful
```

Perhaps documentation at https://deployer.org/docs/7.x/recipe/deploy/rollback should show one example like `dep rollback -o rollback_candidate=46 production` to show destination should be added at the end.
